# Copyright (C) 2018-2019 LEIDOS.
#
# Licensed under the Apache License, Version 2.0 (the "License"); you may not
# use this file except in compliance with the License. You may obtain a copy of
# the License at
#
# http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
# License for the specific language governing permissions and limitations under
# the License.

# Double: The rate at which the Health Monitor Node will process message
# Units: hz
spin_rate_hz: 10.0

# Double: The timeout threshold for essential drivers
# Units: ms
required_driver_timeout: 1500.0

# Double: The time allocated for system startup
# Units: s
startup_duration: 30.0

# List of String: Required plugins for the platform to be functional
required_plugins:
  - Autoware Plugin

# List of String: Required drivers for the platform to be functional
required_drivers:
  - /hardware_interface/ssc_interface_wrapper
  - /hardware_interface/velodyne_lidar_driver_wrapper
